main.js

```diff
@@ -260,12 +267,26 @@ renderApp(app)
 
 $catalogView.addEventListener('click', (event) => {
   var $closestItem = event.target.closest('.card')
-  var clickedItemId = parseInt($closestItem.dataset.itemId, 10)
   if ($closestItem) {
+    var clickedItemId = parseInt($closestItem.dataset.itemId, 10)
     app.view = 'details'
     app.details.item = getItem(app.catalog.items, clickedItemId)
     renderApp(app)
   }
+  else if (event.target.id === 'sort-dropdown') {
+    var $dropdown = document.querySelectorAll('.dropdown-item')
```

Comment:
Instead of reaching into the page, you should augment your application state to account for whether or not to show the dropdown items.